commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -94,7 +95,7 @@ public void testGetStringResourceByName_returnsCorrectString() {
 
         String stringResource = TransferMethodUtils.getStringResourceByName(mContext,
                 HyperwalletTransferMethod.TransferMethodTypes.WIRE_ACCOUNT);
-        assertThat(stringResource, is("wire acc"));
+        MatcherAssert.assertThat(stringResource, Matchers.is("wire acc"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
rmeoved